scripts/installer_test.go

```diff
@@ -118,6 +118,8 @@ func TestInstaller_Config(t *testing.T) {
 }
 
 func TestUpgrade_Config(t *testing.T) {
+	t.Skipf("upgrade test requires binaries to have greater version than \"old release\"")
```

Comment:
I'm not sure whats the issue here

Reply:
Upgrade fails if the "new" msi is for an older version. Currently, unless HEAD is tagged with a version number, it's version is 0.0.0 which causes the upgrade test to fail when it uses a release msi for the "old" installation.